From: Bernhard Walle <bwalle@suse.de>
Subject: [PATCH] Strip L2^B symbols
Patch-mainline: never
References: bnc #456682

This patches strips all L2^B symbols that happen on s390 only from System.map.
We don't need that symbols as this are local labels. It confuses (older)
versions of crash and just makes System.map larger.

The proper fix needs to be in binutils. However, since the binutils maintainer
at SUSE is not cooperative I workarounded this in the kernel. The proper
binutils patch is already mainline [1].


Signed-off-by: Bernhard Walle <bwalle@suse.de>

[1] http://article.gmane.org/gmane.comp.gnu.binutils.cvs/12731
---
 scripts/mksysmap |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/scripts/mksysmap
+++ b/scripts/mksysmap
@@ -41,5 +41,5 @@
 # so we just ignore them to let readprofile continue to work.
 # (At least sparc64 has __crc_ in the middle).
 
-$NM -n $1 | grep -v '\( [aNUw] \)\|\(__crc_\)\|\( \$[adt]\)' > $2
+$NM -n $1 | grep -v '\( [aNUw] \)\|\(__crc_\)\|\( \$[adt]\)\|\(L2\)' > $2
 
